install-spring-boot-on-an-oracle-instance.md

```diff
@@ -22,19 +22,19 @@ redirect: https://developer.oracle.com/tutorials/install-spring-boot-on-an-oracl
 ---
 {% slides %}
 
-In this tutorial, you'll use an Oracle Cloud Infrastructure Free Tier account to set up an Oracle Linux compute instance. Then, install a Spring Boot application and access your new app from the internet. Finally, this tutorial covers all the steps necessary to set up a virtual network for your host and connect the host to the internet.
+In this tutorial, you'll use an Oracle Cloud Infrastructure Free Tier account to set up an Oracle Linux compute instance, install a Spring Boot application, and then configure it to be accessible from the internet. This tutorial also covers all the steps necessary to set up a virtual network for your host and connect that host to the internet.
 
 Key tasks include how to:
 
 * Set up a compartment for your development work.
-* Install your Oracle Linux instance and connect it to your Virtual Cloud Network (VCN). 
-    * Set up an Oracle Cloud Infrastructure virtual cloud network and related network services required for your host to connect to the internet.
-    * Set up `ssh` encryption keys to access your Oracle Linux Server.
+* Install your Oracle Linux instance and connect it to your Virtual Cloud Network (VCN).
+  * Set up an Oracle Cloud Infrastructure virtual cloud network and related network services required for your host to connect to the internet.
+  * Set up `ssh` encryption keys to access your Oracle Linux Server.
```

Comment:
List items should be indented 4 characters. This is for compatibility with ALL flavors of Markdown, even though 2 would work in most cases. 4 will ALWAYS work.

Reply:
Interesting. The linter didn't flag that as an issue so I didn't change the existing list formatting. But I'll definitely make sure to check that in the future!

Reply:
Updated